src/test/java/uk/gov/hmcts/probate/service/NotificationServiceTest.java

```diff
@@ -401,4 +402,227 @@ void sendStopResponseReceivedEmailThrowsExceptionWhenEmailIsNull() throws Notifi
         assertEquals("Email address not found for StopResponseReceivedEmail case ID: 12345",
                 exception.getMessage());
     }
+
+    @Test
+    void updatePersonalisationText_ForGrantIssuedCaseTypeGop() {
+        CaseData caseData = mock(CaseData.class);
+        CaseDetails caseDetails = mock(CaseDetails.class);
+        when(caseDetails.getData()).thenReturn(caseData);
+        when(caseData.getCaseType()).thenReturn("gop");
+        when(caseData.getApplicationType()).thenReturn(ApplicationType.SOLICITOR);
+        when(caseData.getLanguagePreference()).thenReturn(LanguagePreference.ENGLISH);
+        when(caseData.getSolsSOTName()).thenReturn("Solicitor Name");
+        when(caseData.getPrimaryApplicantEmailAddress()).thenReturn("primary@probate-test.com");
+        when(caseData.getSolsSolicitorEmail()).thenReturn("abc@gmail.com");
```

Comment:
i think we only actually need:
```suggestion
        CaseData caseData = mock(CaseData.class);
        when(caseData.getCaseType()).thenReturn("gop");
        when(caseData.getLanguagePreference()).thenReturn(LanguagePreference.ENGLISH);
```
here? (and probably below)